tests/posix/headers/src/netinet_in_h.c

```diff
@@ -58,31 +73,31 @@ ZTEST(posix_headers, test_netinet_in_h)
 	zassert_not_equal(-1, IPPROTO_UDP);
 
 	zassert_not_equal(-1, INADDR_ANY);
-	/* zassert_not_equal(-1, INADDR_BROADCAST); */ /* not implemented */
+	zassert_equal(0xffffffff, INADDR_BROADCAST);
```

Comment:
I'd recommend using `zexpect_equal` here (and other tests too). This will show if multiple tests would fail, rather than if only the first one.

Reply:
That is a great point, I will fix the tests. I think this pr will go to 4.1, no point trying to get it to 4.0.